AxonDeepSeg/apply_model.py

```diff
@@ -278,9 +277,11 @@ def axon_segmentation(path_acquisitions_folders, acquisitions_filenames, path_mo
             for j in range(n_classes):
                 mask[pred == j] = paint_vals[j]
             # Then we save the image
-            ads.imwrite(path_acquisitions_folders[i] / segmentations_filenames[i], mask, 'png')
+            print(  str(segmentations_filenames[i]))
+            image_name =convert_path(acquisitions_filenames[i]).stem
```

Comment:
```suggestion
            image_name = convert_path(acquisitions_filenames[i]).stem
```

Reply:
Adhere to PEP8 guidelines

Reply:
Thanks @mariehbourget  for reviewing this PR. I might have overlooking the PEP8 guidelines. Fixed in commit b0bc23c.